src/com/company/Main.java

```diff
@@ -12,14 +12,14 @@ public static void main(String[] args){
 
         String txtfile = args[0];
         File file = new File(txtfile);
-        checkforsupport(file);
-        checkforexistance(file);
+        checkForSupport(file);
```

Comment:
This method is a void method and if the file does not exists also it just prints "file does not exists" and program execution continues right? You might want to take care of this? Same with checkForSupport  and checkForExistence too

Reply:
i have  used return statement to terminate the execution if it prints "file does not exists" statement